.zenodo.json

```diff
@@ -0,0 +1,34 @@
+{
+    "creators": [
+        {
+            "orcid": "0000-0001-6262-647X",
+            "affiliation": "University of Granada, CERN",
```

Comment:
I think this will be shown as a single affiliation. Does `"University of Granada", "CERN"` would work ?